src/world-company-remuneration/salary-calculator-service-impl.ts

```diff
@@ -3,6 +3,20 @@ const K = "K";
  * @author Jack Napier <jnapier@therealjoker.com>
  */
 export function salary(a: number, b: number) {
```

Comment:
```suggestion
export function salary(a: number, seniorityInYear: number) {
```
b seems to be the seniority in year according to the specification.
a seems to be the empleeId according to the tests ( ex: SATANSON 😆 = 666)

Reply:
b the seniority in year can be transform to JUNIOR, CONFIRMER, SENIOR in order to have a more explicit code here, example:
type SeniorityStatus = "JUNIOR" | "CONFIRMED" | "SENIOR"

const JUNIOR = { code: JUNIOR min: 0, max: 5 } as const
const CONFIRMED = { code: CONFIRMED, min: 5, max: 10 } as const
const SENIOR =   { code: SENIOR, min: 10 } as const

then in your condition, example :
```
function findSeniorityStatus(seniorityInYear: number) {
  if (JUNIOR.min > seniorityInYear && JUNIOR.max < seniorityInYear) {
     return JUNIOR.code
  } ...etc with CONFIRMED and SENIOR
}

const currentCandidateSeniority = findSeniorityStatus(seniorityInYear) 

if (currentCandidateSeniority === "JUNIOR")....apply salary...etc for CONFIRMED and SENIOR
```

We should definitly talk about it (pairing ?)